packages/api/amplify_api/example/integration_test/graphql/api_key_test.dart

```diff
@@ -164,6 +165,38 @@ void main({bool useExistingTestUser = false}) {
             await expectLater(dataCompleter2.future, completes);
           });
 
+          testWidgets('should support where clause when using Model helpers',
+              (WidgetTester tester) async {
+            final blogName =
+                'Integration Test Blog - subscription filter ${uuid()}';
+
+            final postTitle =
+                'Integration Test Post - subscription filter ${uuid()}';
+
+            final blog = await addBlog(blogName);
+
+            // TODO(equartey): Replace this statement when Model object supports parent id field.
+            final pred =
+                const QueryField<String>(fieldName: 'blogID').eq(blog.id);
+
+            final subscriptionRequest = ModelSubscriptions.onCreate(
+              Post.classType,
+              authorizationMode: APIAuthorizationType.userPools,
```

Comment:
nit/question: Why is this test in the api key suite if using userPools? Can it be moved to that suite?